virttest/qemu_devices/qcontainer.py

```diff
@@ -2524,6 +2569,10 @@ def cdroms_define_by_params(self, name, image_params, media=None,
                 scsi_hba = "virtio-scsi-ccw"
         shared_dir = os.path.join(data_dir.get_data_dir(), "shared")
         cache_mode = image_params.get('image_aio') == 'native' and 'none' or ''
+
+        # iso image can be stored in a curl storage, e.g. http server
+        image_access = storage.ImageAccessInfo.access_info_define_by_params(
+            name, image_params)
```

Comment:
Do we need to set a condition `if image_params.get('storage_type') ==  'crul':` ? I am afraid it(`access_info_define_by_params`) will work with other storage types such as ceph, gluster, iscsi which the iso image can not be stored then maybe error in `images_define_by_variables`.

Reply:
We have already disabled cdrom support for other storage backends, but, as we only support a curl(e.g.http/https) cdrom, so it's better to check if it's curl enabled. 
Updated.